packages/suite/src/views/settings/device/DeviceLabel.tsx

```diff
@@ -26,9 +20,8 @@ export const DeviceLabel = ({ isDeviceLocked }: DeviceLabelProps) => {
         applySettings: deviceSettingsActions.applySettings,
     });
     const { anchorRef, shouldHighlight } = useAnchor(SettingsAnchor.DeviceLabel);
-    const { translationString } = useTranslation();
 
-    const [label, setLabel] = useState('');
+    const [label, setLabel] = useState<string>('');
```

Comment:
The type can be inferred from the default state, not necessary to be explicit here.